person/src/com/person/core/exceptions/NotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.person.core.exceptions;
+
+public class NotFoundException extends ValidationException {
```

Comment:
@tatevikb jan ValidationException cant be base exception . U should have BaseException from which ValidationException , NotFoundException etc. will extends

Reply:
Ok, I created BaseException. Anyway, I still don't understand why this option is preferable to the previous one․